src/chainstate/burn/db/sortdb.rs

```diff
@@ -3000,10 +3016,56 @@ impl SortitionDB {
         match epoch {
             StacksEpochId::Epoch10 => true,
             StacksEpochId::Epoch20 => {
-                version == "1" || version == "2" || version == "3" || version == "4"
+                version == "1"
+                    || version == "2"
+                    || version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
+            }
+            StacksEpochId::Epoch2_05 => {
+                version == "2"
+                    || version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
+            }
+            StacksEpochId::Epoch21 => {
+                version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
+            }
+            StacksEpochId::Epoch22 => {
+                version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
+            }
+            StacksEpochId::Epoch23 => {
+                version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
+            }
+            StacksEpochId::Epoch24 => {
+                version == "3"
+                    || version == "4"
+                    || version == "5"
+                    || version == "6"
+                    || version == "7"
+                    || version == "8"
```

Comment:
@stjepangolemac I wonder if maybe this version 8 should only be added to epoch24? Would this potentially cause the node to get stuck??

Reply:
Okay will try keeping it in the epoch24 and see what happens

Reply:
It didn't help.